include/znc/Modules.h

```diff
@@ -720,6 +720,10 @@ class CModule {
 	 *  @return See CModule::EModRet.
 	 */
 	virtual EModRet OnUserTopicRequest(CString& sChannel);
+	/** This module hook is called when a user requests to quit from network.
+	 *  @param sMessage The quit message the client sent.
+	 */
+	virtual void OnUserQuit(CString& sMessage);
```

Comment:
Right. But it is QUIT from client, so it is clear client disconnect really soon. (Anyway, I added return value)